test/unit/pusher_test.rb

```diff
@@ -258,7 +258,7 @@ class PusherTest < ActiveSupport::TestCase
         should "be false if not owned by user and an indexed version exists" do
           create(:version, rubygem: @rubygem, number: '0.1.1')
           refute @cutter.authorize
-          assert_equal "You do not have permission to push to this gem.", @cutter.message
+          assert_includes @cutter.message, "You do not have permission to push to this gem."
```

Comment:
Thank you. My thinking was that since this is tested in full in the integration test, then testing it in full in two places makes the test suite more fragile (more things break on a copy change) without increasing coverage. But perhaps it would be better to test those details here in this faster test, and skip those details in the integration test. What do you think?